.vscode/launch.json

```diff
@@ -28,13 +29,19 @@
             "program": "${workspaceRoot}/src/azure-cli/azure/cli/__main__.py",
             "cwd": "${workspaceRoot}",
             "args": [
-                "--help"
+                //"--help"
+                "interactive"
             ],
             "console": "externalTerminal",
             "debugOptions": [
                 "WaitOnAbnormalExit",
                 "WaitOnNormalExit"
-            ]
+            ],
+            "env": {
+                "API_KEY": "",
+                "ENDPOINT": "",
+                "DEPLOYMENT": ""
+            }
         },
         {
             "name": "Azdev Scripts",
```

Comment:
Please note that your local update in config file should not be included in this PR.